File: app/src/main/java/com/yiann0s/retronebefits/utils/Service.java
package com.yiann0s.retronebefits.utils;

import com.yiann0s.retronebefits.model.Car;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface Service {
    @GET("sample_array.json")
    Call<List<Car>> getJson();
}

File: app/src/main/java/com/yiann0s/retronebefits/utils/Service.kt
package com.yiann0s.retronebefits.utils

import com.yiann0s.retronebefits.model.Car
import retrofit2.Call
import retrofit2.http.GET

interface Service {
    @GET("sample_array.json")
    fun getJson() : Call<MutableList<Car>>
}